js/includes/alert.js

```diff
@@ -14,7 +14,19 @@
  */
 function addAlert( message ) {
 	// Add the tribe-marketing-alert class to the body tag so we can style appropriately
-	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-marketing-alert' )
+	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-alert' )
```

Comment:
I don't think we should change this line (should be no need to change what's inside this function). I recommend:
```suggestion
	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-marketing-alert' )
```